src/main/java/org/testng/TestNGAntTask.java

```diff
@@ -392,6 +393,10 @@ public void setMode(Mode mode) {
     this.mode = mode;
   }
 
+  public void setForkJvm(boolean forkJvm) {
```

Comment:
For an unknown reason, output (stdout and stderr) is not working. `forkJvm=false` allows to use break points.